<?xml version="1.0" encoding="utf-8" standalone="no"?>
<!--This file represents the results of running a test suite-->
<test-results name="C:\Documents and Settings\02272453856\Meus documentos\Visual Studio 2005\Projects\TestNUnit\UnitTest\bin\Debug\UnitTest.dll" total="6" errors="0" failures="1" not-run="1" inconclusive="0" ignored="1" skipped="0" invalid="0" date="2015-05-22" time="14:15:00">
  <environment nunit-version="2.6.3.13283" clr-version="2.0.50727.3655" os-version="Microsoft Windows NT 5.1.2600 Service Pack 3" platform="Win32NT" cwd="C:\NUnit-2.6.3\bin" machine-name="SERPRO1462923V1" user="02272453856" user-domain="SERPRO-RLSL" />
  <culture-info current-culture="pt-BR" current-uiculture="pt-BR" />
  <test-suite type="Assembly" name="C:\Documents and Settings\02272453856\Meus documentos\Visual Studio 2005\Projects\TestNUnit\UnitTest\bin\Debug\UnitTest.dll" executed="True" result="Failure" success="False" time="0.475" asserts="0">
    <results>
      <test-suite type="Namespace" name="UnitTest" executed="True" result="Failure" success="False" time="0.467" asserts="0">
        <results>
          <test-suite type="TestFixture" name="NUnit" executed="True" result="Failure" success="False" time="0.304" asserts="0">
            <results>
              <test-case name="UnitTest.NUnit.Addition" executed="True" result="Success" success="True" time="0.019" asserts="1" />
              <test-suite type="ParameterizedTest" name="Divide" executed="True" result="Failure" success="False" time="0.094" asserts="0">
                <results>
                  <test-case name="UnitTest.NUnit.Divide(0)" description="Teste de Divisão por Zero" executed="True" result="Success" success="True" time="0.018" asserts="0" />
                  <test-case name="UnitTest.NUnit.Divide(3)" description="Teste de Divisão" executed="True" result="Success" success="True" time="0.000" asserts="1" />
                  <test-case name="UnitTest.NUnit.Divide(1)" description="Teste de Divisão" executed="True" result="Failure" success="False" time="0.060" asserts="1">
                    <failure>
                      <message><![CDATA[  Expected: 1
  But was:  3
]]></message>
                      <stack-trace><![CDATA[em UnitTest.NUnit.Divide(Int32 divid) na C:\Documents and Settings\02272453856\Meus documentos\Visual Studio 2005\Projects\TestNUnit\UnitTest\UnitTest.cs:linha 40
]]></stack-trace>
                    </failure>
                  </test-case>
                </results>
              </test-suite>
              <test-case name="UnitTest.NUnit.Function" description="Teste de Função" executed="True" result="Success" success="True" time="0.001" asserts="1">
                <categories>
                  <category name="Iteration2" />
                </categories>
              </test-case>
              <test-case name="UnitTest.NUnit.Multiply" description="Multiplicação" executed="True" result="Success" success="True" time="0.000" asserts="1" />
              <test-case name="UnitTest.NUnit.Substraction" executed="False" result="Ignored">
                <reason>
                  <message><![CDATA[Ignore a Test]]></message>
                </reason>
              </test-case>
            </results>
          </test-suite>
        </results>
      </test-suite>
    </results>
  </test-suite>
</test-results>